config/ember.js

```diff
@@ -1,9 +1,18 @@
 module.exports = {
-  extends: require.resolve('./base.js'),
+  extends: [
+    'plugin:ember/recommended',
+    require.resolve('./base.js')
+  ],
   env: {
     'browser': true
   },
   rules: {
-    'new-cap': [ 'error', { 'capIsNewExceptions': [ 'A' ] } ]
+    'new-cap': [ 'error', { 'capIsNewExceptions': [ 'A' ] } ],
+
+    // Custom rules
+    'ember/avoid-leaking-state-in-components': 'error',
```

Comment:
This rule is deprecated and replaced by `avoid-leaking-state-in-ember-objects`, so we shouldn't need it here.

Reply:
Yup, my bad. Forgot about this.